src/safeds/data/tabular/containers/_time_series.py

```diff
@@ -906,3 +907,151 @@ def plot_lagplot(self, lag: int) -> Image:
         plt.close()  # Prevents the figure from being displayed directly
         buffer.seek(0)
         return Image.from_bytes(buffer.read())
+
+    def plot_time_series_lineplot(self, y_column_name: str | None = None, x_column_name: str | None = None) -> Image:
```

Comment:
```suggestion
    def plot_lineplot(self, x_column_name: str | None = None, y_column_name: str | None = None) -> Image:
```

* We already know that we are calling the method on a time series, so there's no reason to repeat that. 
* Specifying x first is more natural. It's also consistent with `Table`

Reply:
changed